pages/algorithms/equivalent-resource.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Equivalent resource 
+key: equivalent-resource 
+---
+
+Non-identical resources can still be equivalent by equally complying to the expectation formed by the user when navigating to them, thus serving an equivalent purpose. This would usually involve that the advertised key content is the same.  
+
+Web pages and documents (e.g. PDFs, office formats etc.) may be equivalent resources, even if the resources:
+* are located on different URLs, including different domains
+* present different navigation options, e.g. through bread crumbs or local sub menus
+* contain different amounts of information and/or differently worded information
+* use different layouts.
+
+If all resources cover the user's expectations created by e.g. the link name equally well, the resources are considered to be equivalent. 
```

Comment:
I don't think you can use "e.g." in the middle of a sentence like this. I think this sentence is a little confusing. I've tried to come up with an alternative, but I'm struggling even trying to understand what you are saying here. It sounds like whether or not something is an equivalent resource depends on the link to that resource? I'm not sure that's how it works.

Reply:
Moved the "e.g. the link name" into a note instead: 

> **Note:** The user's expectations for the resource can be formed by different things, e.g. the name of the link leading to the resource, with or without the context around the link. This depends on the accessibility requirement that is tested.

Does this work for you?

Reply:
LGTM